serverlessworkflow/modules/ROOT/pages/integrations/expose-metrics-to-prometheus.adoc

```diff
@@ -1 +1,123 @@
-//= Exposing the workflow base metrics to Prometheus
\ No newline at end of file
+Exposing the workflow base metrics to Prometheus on Openshift
+=============================================================
+11-03-2022
+:compat-mode!:
+// Metadata:
+:description: Exposing the workflow base metrics to Prometheus
+:keywords: kogito, workflow, quarkus, metrics, prometheus
+// links
+:openshift_micrometer_url: https://quarkus.io/blog/micrometer-prometheus-openshift
+:dashbuilder_url: https://www.dashbuilder.org/
+:grafana_url: https://grafana.com/
+:quarkus_micrometer_url: https://quarkus.io/guides/micrometer
+:openshift_monitoring_url: https://docs.openshift.com/container-platform/4.7/monitoring/enabling-monitoring-for-user-defined-projects.html
+:prometheus_operator_url: https://prometheus-operator.dev/
+
+Kogito Serverless Workflow generate metrics that can be consumed by Prometheus and visualized by dashboard tools such as link:{openshift_micrometer_url}[Openshift], link:{dashbuilder_url}[Dashbuilder] and link:{grafana_url}[Grafana].
```

Comment:
Don't we want also to support on Kubernetes ? Is there much a difference ?

Reply:
The only reason AFAIK is that I didn't try it on a pure Kubernetes installation, only on Openshift